### PR TITLE
added allegations of harm and violence for 2 screens

### PR DESCRIPTION
**Pull request by SasankAchyuta9**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/respondent/task-list/tasklist.ts [hunk 1]
```diff
@@ -107,8 +108,8 @@ const getTheApplicationSection = (taskListItems, userCase: CaseWithId, userIdamI
     itemList.push({
       id: 'check_the_application',
       text: taskListItems.check_the_application,
```
Comment (khurshedk) [inline src/main/steps/respondent/task-list/tasklist.ts:110]: Why are we removing it, can you please explain

File: src/main/steps/tasklistresponse/start/content.test.ts [hunk 2]
```diff
@@ -122,6 +122,17 @@ describe('task-list > content', () => {
           ],
           title: 'Application details',
         },
+        {
+          items: [
+            {
+              id: 'allegations_of_harm_and_violence',
+              text: 'Allegations of harm and violence',
+              status: 'NOT_AVAILABLE_YET',
+              href: '/tasklistresponse/allegations-of-harm-and-violence/safety-concerns-guidance-page',
```
Comment (khurshedk) [inline src/main/steps/tasklistresponse/start/content.test.ts:131]: please place this into Urls.ts file

File: src/main/steps/tasklistresponse/start/utils.ts [hunk 3]
```diff
@@ -214,3 +214,10 @@ export const getYourSafetyStatus = (userCase: Partial<CaseWithId> | undefined):
   }
   return SectionStatus.TO_DO;
 };
+
+export const getAllegationOfHarmStatus = (userCase: CaseWithId): SectionStatus => {
+  if (userCase.yourchildconcernsstart === 'No' || userCase.yourchildconcernsstart === 'Yes') {
```
Comment (khurshedk) [inline src/main/steps/tasklistresponse/start/utils.ts:219]: use constants instead YesOrNo.yes or YesOrNo.No

File: src/main/steps/respondent/task-list/tasklist.ts (hunk 1 above)
Comment (SasankAchyuta9) [reply src/main/steps/respondent/task-list/tasklist.ts:110]: Actually this should be part of response task list . In ticket its mentioned for respondent task list .. so I migrated it to task list response.

File: src/main/steps/tasklistresponse/start/content.test.ts (hunk 2 above)
Comment (SasankAchyuta9) [reply src/main/steps/tasklistresponse/start/content.test.ts:131]: Done Khurshed.

File: src/main/steps/tasklistresponse/start/utils.ts (hunk 3 above)
Comment (SasankAchyuta9) [reply src/main/steps/tasklistresponse/start/utils.ts:219]: Done Khurshed.

**Pull request closed without merging**